### PR TITLE
[FIXED] Able to Change the Values using Upward and Downward arrow

### LINKED ISSUE #7572: Unable to Change the Values using Upward and Downward arrow
Unable to Change the Values using Upward and Downward arrows in the Oxygen Capacity tabs in the Update Facility Page.
Here the only values can be entered but can't be updated using the available arrows with respective of all fields.

image

### PR DESCRIPTION
**Pull request by manasvi-gaur**
- Fixes #7572 
- Now able to Change the Values using Upward and Downward arrow
image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

### CODE REVIEW

File: src/Components/Facility/FacilityCreate.tsx [hunk 1]
```diff
@@ -985,5 +985,5 @@ export const FacilityCreate = (props: FacilityProps) => {
 };
 
 const FieldUnit = ({ unit }: { unit: string }) => {
-  return <p className="mr-8 text-xs text-gray-700">{unit}</p>;
+  return <p className="mr-1 text-xs text-gray-700">{unit}</p>;
```
Comment (bodhish) [inline src/Components/Facility/FacilityCreate.tsx]:
```suggestion
  return <p className="text-xs text-gray-700">{unit}</p>;
```

File: src/Components/Form/FormFields/TextFormField.tsx [hunk 2]
```diff
@@ -100,15 +100,15 @@ const TextFormField = forwardRef((props: TextFormFieldProps, ref) => {
       );
     const _trailing =
       trailing === trailingFocused ? (
-        <div className="absolute inset-y-0 right-0 flex items-center pr-3">
+        <div className="absolute inset-y-0 right-7 flex items-center pr-3">
```
Comment (bodhish) [inline src/Components/Form/FormFields/TextFormField.tsx]:
```suggestion
        <div className="absolute inset-y-0 right-0 md:right-6 flex items-center pr-3">
```

File: src/Components/Form/FormFields/TextFormField.tsx [hunk 3]
```diff
@@ -100,15 +100,15 @@ const TextFormField = forwardRef((props: TextFormFieldProps, ref) => {
...
           {trailing}
         </div>
       ) : (
         <>
-          <div className="absolute inset-y-0 right-0 flex translate-y-0 items-center pr-3 opacity-100 transition-all delay-300 duration-500 ease-in-out peer-focus:translate-y-1 peer-focus:opacity-0">
+          <div className="absolute inset-y-0 right-7 flex translate-y-0 items-center pr-3 opacity-100 transition-all delay-300 duration-500 ease-in-out peer-focus:translate-y-1 peer-focus:opacity-0">
```
Comment (bodhish) [inline src/Components/Form/FormFields/TextFormField.tsx]:
```suggestion
          <div className="absolute inset-y-0 right-0 md:right-6 flex translate-y-0 items-center pr-3 opacity-100 transition-all delay-300 duration-500 ease-in-out peer-focus:translate-y-1 peer-focus:opacity-0">
```

File: src/Components/Form/FormFields/TextFormField.tsx [hunk 4]
```diff
@@ -100,15 +100,15 @@ const TextFormField = forwardRef((props: TextFormFieldProps, ref) => {
...
             {trailing}
           </div>
-          <div className="absolute inset-y-0 right-0 flex -translate-y-1 items-center pr-3 opacity-0 transition-all delay-300 duration-500 ease-in-out peer-focus:translate-y-0 peer-focus:opacity-100">
+          <div className="absolute inset-y-0 right-7 flex -translate-y-1 items-center pr-3 opacity-0 transition-all delay-300 duration-500 ease-in-out peer-focus:translate-y-0 peer-focus:opacity-100">
```
Comment (bodhish) [inline src/Components/Form/FormFields/TextFormField.tsx]:
```suggestion
          <div className="absolute inset-y-0 right-0 md:right-6 flex -translate-y-1 items-center pr-3 opacity-0 transition-all delay-300 duration-500 ease-in-out peer-focus:translate-y-0 peer-focus:opacity-100">
```

**Review by bodhish — CHANGES_REQUESTED** [review]
Lets also make sure that the buttons looks good in mobile;

**Comment by manasvi-gaur** [comment]
@bodhish done

**Comment by nihal467** [comment]
@manasvi-gaur 
image

in the mobile view, the arrow is not visible anymore

**Pull request closed without merging**